addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/OnkyoBindingConstants.java

```diff
@@ -68,6 +69,7 @@
     public static final ThingTypeUID THING_TYPE_TXNR626 = new ThingTypeUID(BINDING_ID, ONKYO_TYPE_TXNR626);
     public static final ThingTypeUID THING_TYPE_TXNR646 = new ThingTypeUID(BINDING_ID, ONKYO_TYPE_TXNR646);
     public static final ThingTypeUID THING_TYPE_TXNR656 = new ThingTypeUID(BINDING_ID, ONKYO_TYPE_TXNR656);
+    public static final ThingTypeUID THING_TYPE_TXNR717 = new ThingTypeUID(BINDING_ID, ONKYO_TYPE_TXNR708);
     public static final ThingTypeUID THING_TYPE_TXNR717 = new ThingTypeUID(BINDING_ID, ONKYO_TYPE_TXNR717);
```

Comment:
Thanks for the PR! Did you also test the changes because the THING_TYPE_TXNR717 is defined here 2 times. So that will not compile. ;-)

Reply:
Thank you for spotting that, I have fixed it now. I havn’t tested it yet. It was taking too long to compile last night so I went to bed.

Reply:
HMM... I made the changes but it still won't compile, do you mind taking another look?

Reply:
According to the Jenkins/Travis pull request build jobs the last changes seem to compile just fine. :smile:

Reply:
Yes, for some reason it said it faild, then i refreshed my browser and it said it passed. Thanks though, Ive tested it now. Seems to be working perfectly, I had reset the receiver before it would connect again with the new binding but other than that its been flawless